code/world.dm

```diff
@@ -18,7 +18,7 @@ var/global/datum/global_init/init = new ()
 	cache_lifespan = 0	//stops player uploaded stuff from being kept in the rsc past the current session
 
 
-#define RECOMMENDED_VERSION 508
+#define RECOMMENDED_VERSION 510 //Goddammit, update this once in a while!

```

Comment:
I don't really feel like it's a shitpost, IMO, it's a comment trying to be
helpful, but whatever you say

On Tue, May 17, 2016, 22:17 Fox McCloud <notifications@github.com> wrote:

> In code/world.dm
> <https://github.com/ParadiseSS13/Paradise/pull/4454#discussion_r63595426>:
>
> > @@ -18,7 +18,7 @@ var/global/datum/global_init/init = new ()
> >  	cache_lifespan = 0	//stops player uploaded stuff from being kept in the rsc past the current session
> >
> >
> > -#define RECOMMENDED_VERSION 508
> > +#define RECOMMENDED_VERSION 510 //Goddammit, update this once in a while!
>
> Drop the shitpost comment, please.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ParadiseSS13/Paradise/pull/4454/files/126c5d679dee628c83c7eac862c5d94ac39a8590#r63595426>
>